<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="browseButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pathBox">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="fileBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
